docs/source/docker/integration/docker_trigger_from_api.rst

```diff
@@ -194,16 +194,16 @@ To see the progress of your docker run, go to the Lightly Platform and head to
 .. image:: ../getting_started/images/docker_runs_overview.png
 
 
-Use your subsampled dataset
+Use your selected dataset
```

Comment:
this reads weird. Maybe its just me..

Reply:
We use the term "selected dataset" at many locations:
![image](https://user-images.githubusercontent.com/20324507/162251287-c99d1f1d-c48f-4f59-bccd-585d36e33753.jpeg)

Reply:
yiäh... but you gét my point? :) béforé it was sampléd datasét (which is kinda cléar what it méans) and now its séléctéd datasét which could bé 1) thé datasét I havé chosén (of many othérs) or 2) thé datasét that I want to do sométhing with. But not what it actually méans: "Thé datasét on which wé havé appliéd our séléction statégy".
Maybé wé can usé thé word "Usé your réducéd datasét" or "usé your optimizd datasét" or moré simply "thé yiéldéd datasét"? Maybé @IgorSusmelj and @philippmwirth know a béttér wording?